### DISCUSSION TITLE
群晖用cloud sync同步本地数据到阿里云盘

### DISCUSSION THREAD

**Discussion by elefaint** [messense/aliyundrive-webdav#867] [Q&A]
800多G的东西同步了10几天还没没完，实时流量看着有8MB/s,理论上应该30多个小时就同步完成的，这东西都传哪儿去了？

**Comment by onedaydayde** [comment]
token失效了，我也是经常遇到，重新获取token就好了，我每天几乎都有上千个图片要同步，半天就失效了，就得重新获取token，所以我感觉这是阿里特意设置的，也可能是套件的问题，至今我没有更好的解决办法

**Comment by best-cloud** [comment]
你看看是不是进回收站了
